scrapers/PornDB.yml

```diff
@@ -11,22 +11,20 @@ xPathScrapers:
     scene:
       URL: //a[@class="btn btn-secondary btn-block"]/@href
       Studio:
-        Name: //div[@class="col-6 text-left scene-page-site mb-4"]/a/p/text()
-      Title: //h3/text()
+        Name: //div[@class="col-6 text-left scene-page-site mb-4"]/a/p/text() #//meta[@property="og:site_name" and @data-vue-meta="1"]/@content
```

Comment:
I think you should be able to drop ` and @data-vue-meta="1"` in all of the optional selectors